server/viame_tasks/tasks.py

```diff
@@ -44,34 +48,50 @@ def run_pipeline(self, path, pipeline):
             )
             image_list_file = temp2.name
         command = [
-            "cd /home/VIAME/build/install/ &&",
+            "cd /opt/noaa/viame &&",
             ". ./setup_viame.sh &&",
-            "pipeline_runner",
-            "-p /home/VIAME/configs/pipelines/{}".format(pipeline),
+            "kwiver",
+            "runner",
+            "-p /opt/noaa/viame/configs/pipelines/{}".format(pipeline),
             "-s input:video_filename={}".format(image_list_file),
             "-s detector_writer:file_name={}".format(output_path),
         ]
```

Comment:
No direct action needed, but for this and the command above, it might be useful to utilize `pathlib`, as these paths are being shared, and it would make this command easier to read.

Reply:
Valid.  I'll do this.

Reply:
Does 27a6e60  resolve your comment?

Reply:
Yep LGTM